core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -278,11 +280,14 @@ public void finished(long requestSize, long responseSize, long latencyMs) {
      * Indicate that a request has failed with an exception.
      */
     public void exception(final RequestEvent event) {
-      final int statusCode = event.getException() instanceof WebApplicationException
-          ? ((WebApplicationException) event.getException()).getResponse().getStatus() : 0;
-      int idx = statusCode / 100;
+      int idx = event.getContainerResponse() != null
```

Comment:
// map the http status codes down to their classes (2xx, 4xx, 5xx)
// use the containerResponse status as it has the http status after ExceptionMappers are applied